pyccel/errors/messages.py

```diff
@@ -60,6 +60,9 @@
 INDEXED_TUPLE = 'Tuples must be indexed with constant integers for the type inference to work'
 LIST_OF_TUPLES = 'Cannot create list of non-homogeneous tuples'
 
+UNDEFINED_DECORATOR = 'Decorator not used'
+UNDEFINED_DECORATORS = 'Decorators not used'
```

Comment:
I'm not sure it's worth having 2 error messages just for this, I would just use:
`Decorator(s) not used`